service/src/main/resources/config/aou/test/SC2023Q3R1/SC2023Q3R1.json

```diff
@@ -30,6 +30,8 @@
     "t_device_occurrence.json",
     "t_heart_rate_level.json",
     "t_heart_rate_summary.json",
+    "t_icd9cm.json",
+    "t_icd9proc.json",
```

Comment:
Did you mean to also add the icd10 entities + groups to this top-level file?

Reply:
good catch! I had it in my local and did not checkin. Checked them in. already built both icd10cm and icd10pcs and tested before creating the PR :-)